README.md

```diff
@@ -5,13 +5,12 @@ A collection of templates that can be used to leverage Delphix Compliance Servic
 To use Delphix Compliance Services for Azure, and specifically to leverage these pipelines you will need to have the
 following linked services in your data factory:
 * REST service for talking to DCS for Azure
-* Azure SQL Database for storing metadata about the data you wish to profile and mask
+* Azure SQL Database for storing metadata about the data you have discovered and rules for masking
 
 ### To Import Latest Version Templates
 
-Run `docker-compose -f docker-compose.yaml up && stty sane`, this will create the latest version of all templates
-and put them in the `releases` directory. The `stty sane` is to fix the terminal as docker-compose doesn't always
-clean up after itself correctly. From there, you can import the template into your data factory using the Data
+Run `docker-compose -f docker-compose.yaml up`, this will create the latest version of all templates
+and put them in the `releases` directory. From there, you can import the template into your data factory using the Data
```

Comment:
Yeah. I'm reasonably confident it was because my install of docker was incorrect on the new Mac hardware, since I ported it from the old hardware Mac and that's when this started.